### ISSUE TITLE
NTR: Dihydroxyphenylacetic acid in blood, urine, and CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8316]
New term request Dihydroxyphenylacetic acid in blood, urine, and CSF (elthat_130220210432): DOPAC; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
since this is labeled blood chemistry, is it sufficient to add a term for increased blood concentration?
I found a usage here -- https://omim.org/entry/223360?search=Dihydroxyphenylacetic&highlight=dihydroxyphenylacetic
but none for urine/CSF

**Comment by pnrobinson** [comment]
Also, is this 
[(3,4-dihydroxyphenyl)acetic acid](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_41941&lang=en&viewMode=All&siblings=false)

**Comment by MickeySegal** [comment]
For CSF: https://www.ncbi.nlm.nih.gov/books/NBK1474/ "High plasma & CSF concentrations of DOPA, DOPAC, & DA"

I no longer see mention of urine in any of our citations.  The CSF is mentioned just once, but it is more practical to use blood so I'm removing  urine and CSF from the name of our finding.

**Comment by MickeySegal** [comment]
Yes, DOPAC = 3,4-dihydroxyphenylacetic acid; I'm changing our finding name accordingly.

**Comment by pnrobinson** [comment]
@MickeySegal 
added as

Adding new term: 
- ID: HP:6000234
- label: Elevated circulating dihydroxyphenylacetic acid concentration
- parents: [Abnormal circulating organic amino compound concentration (HP:0033405)](https://hpo.jax.org/app/browse/term/HP:0033405)
- pmid: PMID:11857564
- def: The concentration of dihydroxyphenylacetic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8316
- synonyms: Elevated circulating DOPAC concentration(NONE)
